src/index.html

```diff
@@ -1,13 +1,373 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="shortcut icon"
+      href="images/favicon.ico"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body id="home" class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+
+            <a href="#home" class="logo">
+              <img
+                class="logo__image"
+                src="./images/icons/logo.svg" alt="Bose">
+            </a>
+
+            <div class="header__top-icon">
+
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone">
+                <div class="icon__tooltip">
+                  +1 234 555-55-55
+                </div>
```

Comment:
you can remove this hover effect that on hover phone number appears for mobile mode as we could not hover on mobile practically
<img width="356" alt="image" src="https://user-images.githubusercontent.com/100297341/193539596-b45ba5a2-fbdf-4419-984d-98736cf41d84.png">